tests/ext/drv/test_ext.c

```diff
@@ -191,6 +193,13 @@ NTSTATUS
 _test_ebpf_extension_hook_provider_detach_client(_In_ void* provider_binding_context);
 
 // Test eBPF extension Hook NPI provider characteristics
+ebpf_attach_provider_data_t _test_ebpf_extension_attach_provider_data = {
+    {0xf788ef4a, 0x207d, 0x4dc3, {0x85, 0xcf, 0x0f, 0x2e, 0xa1, 0x07, 0x21, 0x3c}}};
```

Comment:
this becomes a dupe of EBPF_PROGRAM_TYPE_TEST then?
If you are getting compiler errors referring to EBPF_PROGRAM_TYPE_TEST,
Maybe make `ebpf_program_type_t* supported_program_type;` a pointer and then initialize _test_ebpf_extension_hook_provider_data with  EBPF_PROGRAM_TYPE_TEST